src/components/common/UploadDocs.js

```diff
@@ -18,6 +21,10 @@ const UploadDocs = ({
   submitButtonClassname,
   apiAxios,
   submissionId,
+  storyId,
+  setHasWritten,
+  setHasDrawn,
```

Comment:
We can pass a reference to a function instead of passing both functions in and conditionally calling them!